src/client/src/containers/ViewLicensesModal/index.tsx

```diff
@@ -28,23 +30,38 @@ const messages = defineMessages({
   closeModalLabel: {
     id: "viewLicensesModal.closeModalLabel",
     defaultMessage: "Close"
+  },
+  licenses: {
+    id: "licenses.licenses",
+    defaultMessage: "Licenses"
   }
 });
 
 const ViewLicensesModal = ({ intl, closeModal }: Props) => {
   const { formatMessage } = intl;
 
+  const cancelKeyDownHandler = (event: React.KeyboardEvent<SVGSVGElement>) => {
+    if (event.key === KEY_EVENTS.ENTER || event.key === KEY_EVENTS.SPACE) {
```

Comment:
Just curious, I was pressing enter/space, but nothing happen. How could I test this code?

Reply:
1) open the licenses modal
2) Click on it (the license modal)
3) Press tab until you highlight the "X" on the top right (the close svg)
4) Press enter/space
5) Observe the modal closing.

Hope this helps!

Reply:
Thanks!